bloom/commands/release.py

```diff
@@ -630,18 +625,6 @@ def get_gh_info(url):
     return url_paths[1], url_paths[2], url_paths[3], '/'.join(url_paths[4:])
 
 
-def create_fork(org, repo, user, password):
-    msg = "Creating fork: {0}:{1} => {2}:{1}".format(org, repo, user)
-    info(fmt("@{bf}@!==> @|@!" + str(msg)))
-    headers = {}
-    headers["Authorization"] = "Basic {0}".format(base64.b64encode('{0}:{1}'.format(user, password)))
-    conn = HTTPSConnection('api.github.com')
-    conn.request('POST', '/repos/{0}/{1}/forks'.format(org, repo), json.dumps({}), headers)
-    resp = conn.getresponse()
-    if str(resp.status) != '202':
-        error("Failed to create fork: {0} {1}".format(resp.status, resp.reason), exit=True)
```

Comment:
I could find no reference to that code in the bloom repo. If you are concerned that an external package might be referencing that code I can restore it -- but I'll also have to update it to use the urllib instead of HTTPSConnection. Let me know if you think that is required.